beeai/README.md

```diff
@@ -12,7 +12,8 @@ Three agents process tasks through Redis queues:
 
 ## Setup
 
-Copy the `templates` directory to `.secrets` and fill in required information.
+- Copy the `templates` directory to `.secrets` and fill in required information.
+    - For `.secrets/rhel-config.json`, you can also copy the content from Bitwarden
```

Comment:
TODO: store this in Bitwarden or jotnar repo?

Reply:
Having it in a git repo will make it easier to update (with a history), so I would say it depends on when this will be replaced by a proper solution querying PP - if there is a chance we will be using it for a while I would vote for storing it in the jotnar repo.